sdk/python/feast/feature_store.py

```diff
@@ -167,7 +169,8 @@ def __init__(
 
         registry_config = self.config.get_registry_config()
         if registry_config.registry_type == "sql":
-            self._registry = SqlRegistry(registry_config, None)
+            self._registry = SqlRegistry(registry_config, None, is_feast_apply=is_feast_apply)
+            self._registry.refresh()
```

Comment:
@IsraelAgundis Why not just call `refresh` in the constructor after setting `_in_feast_apply_context`?

Reply:
Good catch.